guides/source/active_storage_overview.md

```diff
@@ -120,7 +120,7 @@ amazon:
   bucket: ""
 ```
 
-Optionally provide a Hash of upload options:
+Optionally provide a Hash of client and upload options:
```

Comment:
One more nitpick I just realized (sorry!): the "Hash" in this sentence was referring to the value of `upload` (e.g. `{ server_side_encryption: "" }`).  I think we can just remove it for clarity:

```suggestion
Optionally provide client and upload options:
```

Reply:
no problem, committed the suggestion